mako.5.scd

```diff
@@ -388,6 +392,25 @@ specifiers.
 
 *%t*	Total number of notifications
 
+# MODES
+
+mako supports applying style options conditionally via modes. A configuration
+section with a _mode_ criteria will only be applied if the current mode
+matches. **makoctl**(1) can be used to change the current mode.
+
+The default mode is named "default".
+
+For example, to hide all notifications if the mode "dnd" (Do Not Disturb) is
```

Comment:
Whoops, forgot to add this comment. I'm just a *tad* worried that "Do Not Disturb" makes it seem like this is a dedicated mode when it's just an arbitrary name. I think removing the clarification or using `"custom"` would be better.

Reply:
Renamed the mode to `do-not-disturb`.